amnestywebsite/humanity-theme
# Adds the 50/50 text with image block pattern

Ref: https://github.com/amnestywebsite/humanity-theme/issues/176

Adds the 50/50 text with image block pattern, and all it's variations (4 variations, a combo white/grey background, left/right text)

**Steps to test**:
1. Edit a post
2. Open the block inserter and check the new patterns appears under the patterns tab
3. Insert the patterns and check it matches the examples in the ticket

Example: http://bigbite.im/i/JuWnxD
